library/Icingadb/Widget/ItemList/BaseDowntimeListItem.php

```diff
@@ -131,23 +132,28 @@ protected function assembleTitle(BaseHtmlElement $title)
             $link = $this->createServiceLink($this->item->service, $this->item->service->host, true);
         }
 
-        if ($this->getNoSubjectLink()) {
-            $title->addHtml(new HtmlElement(
-                'span',
-                Attributes::create(['class' => 'subject']),
-                Text::create($this->item->is_flexible ? t('Flexible Downtime') : t('Fixed Downtime'))
-            ));
+        if ($this->item->is_flexible) {
+            $template = '{{#link}}Flexible Downtime{{/link}}';
         } else {
-            $title->addHtml(new Link(
-                $this->item->is_flexible
-                    ? t('Flexible Downtime')
-                    : t('Fixed Downtime'),
-                Links::downtime($this->item)
-            ));
+            $template = '{{#link}}Fixed Downtime{{/link}}';
         }
 
         if ($link !== null) {
-            $title->addHtml(Text::create(': '), $link);
+            $template .= ' for %s';
+        }
+
+        if ($this->getNoSubjectLink()) {
+            $title->addHtml(TemplateString::create(
+                t($template),
```

Comment:
Localizable strings **must** be complete. They **must not** be variables, only strings are supported.

`t('foo')` is fine, `t($foo)` not. `t('abcd' . 'efgh')` is fine, `t('abcd' . $efgh)` not. Oh, and `t('abcd' . $bla ? 'blub' : 'blubber')` is also not supported.

Reply:
✅